wolfcrypt/src/ecc.c

```diff
@@ -7654,7 +7654,10 @@ int wc_ecc_free(ecc_key* key)
     mp_clear(key->pubkey.y);
     mp_clear(key->pubkey.z);
 
-    mp_forcezero(key->k);
+#ifdef ALT_ECC_SIZE
```

Comment:
The primary issue is `fp_forcezero` does not NULL check. You are correct though only ALT_ECC_SIZE can have a pointer here. I would recommend changing both.

Reply:
Good point, looks like integer and SP math have this check, only TFM doesn't.  Fixed.